apps/api/test/functional/wallets-refill.spec.ts

```
@@ -1,85 +1,80 @@
-import { app } from "@src/rest-app";
+import { ManagedUserWalletService } from "@src/billing/services";
+import { CORE_CONFIG } from "@src/core";
+import type { ApiPgDatabase } from "@src/core";
+import { POSTGRES_DB, resolveTable } from "@src/core";
+import { UserRepository } from "@src/user/repositories";
 
-import { topUpWallet } from "@test/services/topUpWallet";
-import { WalletTestingService } from "@test/services/wallet-testing.service";
+import { createAkashAddress } from "@test/seeders/akash-address.seeder";
+import { DeploymentGrantResponseSeeder } from "@test/seeders/deployment-grant-response.seeder";
+import { FeeAllowanceResponseSeeder } from "@test/seeders/fee-allowance-response.seeder";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)